packages/react-scripts/template/README.md

```diff
@@ -162,6 +163,14 @@ Instead, it will copy all the configuration files and the transitive dependencie
 
 You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
 
+## Supported Features Beyond ES6
+
+The following language features beyond ES6 are supported (at the time of writing, stage 2 and 3 proposals):
+
+* [Public class fields](https://github.com/babel/babel/tree/master/packages/babel-plugin-transform-class-properties) ([stage 2](https://tc39.github.io/proposal-class-public-fields/))
```

Comment:
Can we link to babel's website docs instead?

http://babeljs.io/docs/plugins/transform-class-properties/
http://babeljs.io/docs/plugins/transform-object-rest-spread/